dns/perf/params/default.yaml

```diff
@@ -0,0 +1,15 @@
+# Default parameters that explores performance space given limited CPU.
```

Comment:
We do not set resource limits -- however, settings limits lets us simulate cases where other load pushes the DNS server to be constrained to its resource request. We want to know under these stress conditions what behavior we can expect from the system.